RFC-009-SWAP-Basic-ERC20.adoc

```diff
@@ -136,18 +139,40 @@ check_expiry:
     refund
     jumpi
 
-exit:
-    // Exit
-    return(0, 0)
+    // return "tooEarly" = 0x746f6f4561726c79000000000000000000000000000000000000000000000000
+    mstore(0, "tooEarly")
+    revert(0, 32)
 
+
+/*
+    memory  layout
+    0 secret
+    32 hash return value
+    ->
+    0 transfer pointer
+    32 to
+    64 amount
+    96 transfer return
+
+    place holders
+    0x3000000000000000000000000000000000000003 // redeem address
+    0x4000000000000000000000000000000000000004 // refund address
```

Comment:
Where is ` 0x5000000000000000000000000000000000000005`? Is this a typo or is it actually 03, 04, 06, 07?

Reply:
no bug, feature (0x5... is the placeholder for the amount) :)
However, let me remove this from the RFC. no need to have this comment in here.